<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paint v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img/img.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="imageLabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="curveBrush"/>
   <addaction name="airbrush"/>
   <addaction name="separator"/>
   <addaction name="circleBrush"/>
   <addaction name="polygonBrush"/>
   <addaction name="separator"/>
   <addaction name="contour"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="dockLayot">
     <item>
      <widget class="QFrame" name="staticWidget">
       <property name="maximumSize">
        <size>
         <width>282</width>
         <height>349</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="currentBrush">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="brushInfo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="open">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="curveBrush">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/curve.png</normaloff>:/img/curve.png</iconset>
   </property>
   <property name="text">
    <string>Curve brush</string>
   </property>
  </action>
  <action name="circleBrush">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/circle.png</normaloff>:/img/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle brush</string>
   </property>
  </action>
  <action name="polygonBrush">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/polygon.png</normaloff>:/img/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon brush</string>
   </property>
  </action>
  <action name="airbrush">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/airbrush.png</normaloff>:/img/airbrush.png</iconset>
   </property>
   <property name="text">
    <string>Airbrush</string>
   </property>
  </action>
  <action name="contour">
   <property name="icon">
    <iconset resource="img/img.qrc">
     <normaloff>:/img/contourSmile.png</normaloff>:/img/contourSmile.png</iconset>
   </property>
   <property name="text">
    <string>Contour animal</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
